### PR TITLE
build: Update JS dependencies

### PR DESCRIPTION
**Pull request by seakayone**
- **build(deps): Update js dependencies and yarn package manager**
- **remove baseUrl**

This only updates some smaller dependencies. I attempted quickly to update Svelte to the latest version but it seems to be a bigger task, hence I have created a followup issue (DEV-4079).

### CODE REVIEW

File: web-frontend/src/store.ts [hunk 1]
```diff
@@ -16,17 +16,10 @@ export const isTestEnvironment = readable(
     window.location.hostname.startsWith('meta.dev'),
 );
 
-export function baseUrl() {
```
Comment (mpro7) [inline web-frontend/src/store.ts:19]: Why was it removed? AFAIR In old setup it was required for working in between different environments. There are no tests here, so could you confirm that mets is working as expected?

**Review by mpro7 — COMMENTED** [review]
Looks fine, but need one confirmation, so only comments for now. Please see below.

File: web-frontend/src/store.ts (hunk 1 above)
Comment (seakayone) [reply web-frontend/src/store.ts:19]: It essentially does not do anything useful, I am not sure why it used to be necessary.

If `fetch` requests start with a `/` then the url will result in the exact same `protocol://host:port` as this function is calculating.

Speaking about tests... I have not found any js test and also no setup for running js tests in the CI. Did I miss anything?

Comment (mpro7) [reply web-frontend/src/store.ts:19]: There is no tests at all on the FE side, this is what I meant.

**Review by mpro7 — APPROVED** [review]

File: web-frontend/src/store.ts (hunk 1 above)
Comment (seakayone) [reply web-frontend/src/store.ts:19]: Adding a test setup is beyond the scope of this PR.

Comment (mpro7) [reply web-frontend/src/store.ts:19]: Great, because I didn't event think about it to add it in this PR, neither proposed it.

**Pull request merged**